src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -339,12 +348,49 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 									))}
 								</Flex>
 							),
-							Empty:
+							Empty: !isNLPChecked ? (
 								!!value && !suggestions?.length ? (
 									<Flex className="not-found-container">
 										<NotFoundCard />
 									</Flex>
 								) : null
+							) : (
```

Comment:
Probably we don't need examples here again. We can only show examples when the search input is empty

Reply:
I put this because we cant disable the autocomplete dropdown while typing, so while typing the examples would show up with the prototype message. I found this to be the best tradeoff since we can't dynamically move the toggle with the drop down results.